src/backend/api/Fusion.Resources.Api/Controllers/Projects/ApiModels/ApiProject.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Fusion.Resources.Domain;
+using System;
+
+namespace Fusion.Resources.Api.Controllers.Projects
+{
+    public class ApiProject
```

Comment:
There exists another api model for project, however here we use the org id as primary id, and `InternalId` for the internal id. 
Think this was done as it's the org project id that is expected in the endpoints?

Maybe align or move the model to the common folder, for consistency?

Reply:
Good catch, moved and reused.